credentialdigger/client.py

```diff
@@ -447,6 +451,27 @@ def update_discovery(self, query, discovery_id, new_state):
 
         return self.query_check(query, new_state, discovery_id)
 
+    def update_discoveries(self, query, discoveries_ids, new_state):
+        """ Change the state of multiple discoveries.
+
+        Parameters
+        ----------
+        discoveries_ids: list
+            The ids of the discoveries to be updated
+        new_state: str
+            The new state of these discoveries
```

Comment:
here you miss the parameter `query`

Reply:
@marcorosa parameter `query` is missing in all methods of `client.py`.  Shall I add it to each method?

Reply:
yes, please